packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -634,6 +636,57 @@ export const RemoteServerFactory = {
         })
     },
 
+    /**
+     * @private
+     */
+    _setupExtensions(app, options) {
+        const extensions = options.mobify?.app?.extensions || []
+        app.__extensions = extensions || []
+
+        let _require
+
+        extensions.forEach((extension) => {
+            const setupServerFilePath = path.join(
+                options.buildDir,
+                'extensions',
+                extension,
+                'setup-server.js'
+            )
+
+            // Only eval when there are extensions
+            // this makes it slightly faster for projects that
+            // have no extensions
+            if (!_require) {
+                _require = eval('require')
+            }
+
+            let setupServer
+
+            try {
+                setupServer = _require(setupServerFilePath)
+            } catch (e) {
+                if (e.message && e.message.startsWith('Cannot find module')) {
+                    // skip extensions that don't have a setup-server.js file
+                    return
+                }
```

Comment:
It's perfectly OK to have an extension with no `setup-server` file, so good choice here.

Reply:
Out of curiosity, what other kind of errors would be thrown during the requiring of the file?

Reply:
the answer is... i don't know! 🤣  i'm just being extra cautious here since if this fails, sites will be completely broken. So i want to make sure i catch all possible exceptions.